src/main/webapp/test/mentors.test.js

```diff
@@ -0,0 +1,42 @@
+import * as Mentors from '../mentors.js';
+import '@testing-library/jest-dom/extend-expect';
+import {getByText} from '@testing-library/dom';
+
+const mockMentor = {
+  interestTags: ['interest1', 'interest2'],
+  projects: [{respositoryID: 'mockRepoID', name: 'mockProjectName'}],
+  bio: 'This is the mock bio.',
+  email: 'samialves@google.com',
+  gitHubID: 'mockGitID',
+  image: '../images/dior.jpg',
+  location: 'Mockville, Mocktana',
+  name: 'Mocky the Mockster',
+};
+
+describe('Mentor card', () => {
+  it('is created with all expected componenets', () => {
```

Comment:
Is it possible to make smaller test blocks?
```
it("smaller description blocks might improve debugging")
```

Reply:
Sounds good, I can seperate by element.